verticapy/vdataframe.py

```diff
@@ -8783,6 +8790,67 @@ def to_vdf(self, name: str):
         file.close()
         return self
 
+    # ---#
+    def train_test_split(
```

Comment:
It make sense to have the train_test_split function in vDataFrame. Can you please add unit tests for it?

Reply:
Yes, it is much more easier and it is part of the data preparation. It allows a simple call with much less parameters. As we have this seeded_random, we can now return vDataFrames without creating temporary elements.

Reply:
Cool!
If you agree, let's treat it as a new feature for vDF. Then, I'll approve the PR after you add tests for it.